frontend/src/pages/comparisons/Comparison.tsx

```diff
@@ -54,26 +56,66 @@ const displayWeeklyCollectiveGoal = (
 const ComparisonPage = () => {
   const { t } = useTranslation();
 
-  const location = useLocation();
-  const searchParams = new URLSearchParams(location.search);
-  const series: string = searchParams.get('series') || 'false';
-  const isEmbedded = Boolean(searchParams.get('embed'));
-
   const {
     options,
     baseUrl,
     active: pollActive,
     name: pollName,
   } = useCurrentPoll();
 
+  const location = useLocation();
+  const searchParams = new URLSearchParams(location.search);
+  const isEmbedded = Boolean(searchParams.get('embed'));
+
+  const [isLoading, setIsLoading] = useState(true);
+  const [comparisonsCount, setComparisonsCount] = useState(0);
+  const [userComparisons, setUserComparisons] = useState<string[]>();
+
+  useEffect(() => {
+    async function getUserComparisonsAsync(
+      pName: string
+    ): Promise<[number, string[]]> {
+      const paginatedComparisons = await getUserComparisonsRaw(pName, 20);
+      let results: string[] = [];
+
+      if (paginatedComparisons.results) {
+        results = paginatedComparisons.results.map(
+          (c) => c.entity_a.uid + '/' + c.entity_b.uid
+        );
+      }
+
+      return [paginatedComparisons.count ?? 0, results];
+    }
+
+    getUserComparisonsAsync(pollName)
+      .then((results) => {
+        setComparisonsCount(results[0]);
+        setUserComparisons(results[1]);
+        setIsLoading(false);
+      })
+      .catch(() => {
+        setIsLoading(false);
```

Comment:
In case of API error, do not display the tutorial.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1684/commits/5e3e351c32ca432572e25e46786d3c7b623e29be